Makefile.core.mk

```diff
@@ -38,6 +38,8 @@ containers:
 	@cd docker/build-tools && ./build-and-push.sh
 
 containers-test:
+	# Workaround https://github.com/kubernetes/test-infra/issues/23741. Should be removed once its fixed upstream
+	@iptables -t mangle -A POSTROUTING -p tcp --tcp-flags SYN,RST SYN -j TCPMSS --clamp-mss-to-pmtu
```

Comment:
I don't think its sufficient to do it in the test, the postsubmit job will fail. I would do it in build-and-push